app/views/measures/grouped/_uk.html.erb

```diff
@@ -1,3 +1,18 @@
+<!-- UK IMPORT CONTROLS -->
+<% if (collection = uk_declarable.import_measures.import_controls.for_country(@search.country)).present? %>

```

Comment:
I think Mr. Demeter is not happy here,
I know you did not create it, but still, it would be nice to not have this loooong chain here.

Reply:
Happy to refactor it and I have a notion of how, actually

Reply:
okay, it's a little bit painful looking into it